source/install/requirements.rst

```diff
@@ -55,8 +55,8 @@ Mobile Apps
 .. csv-table::
     :header: "Operating System", "Technical Requirement"
 
-    "iOS", "iPhone 5s devices and later with iOS 11+"
-    "Android", "Android devices with Android 7+"
+    "iOS", "iPhone 5s devices and later with iOS 12+"
```

Comment:
We need to look up how this works for iOS (does the app store handle serving the correct build for the OS, how do we specificy this in the store? etc)

Reply:
https://apple.stackexchange.com/questions/254172/how-can-i-make-allow-users-with-older-versions-of-ios-to-download-an-older-versi

Reply:
can we remove `iPhone 5s` and change it for something like `Apple devices with iOS 12+` ?